### PR TITLE
Add some uts for setting bpf log level

### PR DESCRIPTION
**Pull request by Okabe-Rintarou-0**
**What type of PR is this?**
enhancement

**What this PR does / why we need it**:
Add some uts for setting bpf log level
**Which issue(s) this PR fixes**:
Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note

```

### CODE REVIEW

File: pkg/status/status_server_test.go [hunk 1]
```diff
@@ -74,6 +78,68 @@ func TestServer_getLoggerLevel(t *testing.T) {
...
+				loggerInfo := LoggerInfo{
+					Name:  bpfLoggerName,
+					Level: testLoggerLevel,
+				}
+				reqBody, _ := json.Marshal(loggerInfo)
+				req := httptest.NewRequest(http.MethodPost, setLoggerUrl, bytes.NewReader(reqBody))
+				w := httptest.NewRecorder()
+				server.setLoggerLevel(w, req)
+
+				if w.Code != http.StatusOK {
```
Comment (LiZhenCheng9527) [inline pkg/status/status_server_test.go:128]: nit: Recommended use of assert.Equal

File: pkg/status/status_server_test.go [hunk 2]
```diff
@@ -74,6 +78,68 @@ func TestServer_getLoggerLevel(t *testing.T) {
...
+					t.Errorf("Expected status code %d, but got %d", http.StatusOK, w.Code)
+				}
+
+				server.bpfLogLevelMap.Lookup(&key, &actualLoggerLevel)
+
+				if actualLoggerLevel != expectedLoggerLevel {
```
Comment (LiZhenCheng9527) [inline pkg/status/status_server_test.go:134]: ditto

File: pkg/status/status_server_test.go [hunk 3]
```diff
@@ -74,6 +78,68 @@ func TestServer_getLoggerLevel(t *testing.T) {
...
+					t.Errorf("Wrong logger level, expected %d, but got %d", expectedLoggerLevel, actualLoggerLevel)
+				}
+			}
+		}
+		cleanup()
```
Comment (hzxuzhonghu) [inline pkg/status/status_server_test.go:139]: So cleanup can not be called if error occurs above. maybe use t.Run(), we can use defer then

**Review by hzxuzhonghu — APPROVED** [review]
/lgtm

Thank you!

**Pull request merged**